lib/vmdb/settings/validator.rb

```diff
@@ -167,6 +167,42 @@ def smtp(data)
 
         return valid, errors
       end
+
+      def workers(data)
+        valid, errors = true, []
+        # worker_settings expects a hash-like structure with nested keys: :config and :workers
+        data = {:config => {:workers => data}}
```

Comment:
Should this be `data = {:config => {:workers => data.to_hash}}` ?  Only asking because I think data is an OpenStruct here.

Reply:
😆   I'm glad you noticed this.  I had `data.to_hash` before, then with testing, it worked as a `Config::Options` since it acts like a hash, so I removed it.